### PR TITLE
basic /chain endpoints

### PR DESCRIPTION
**Pull request by fboucquez**
New basic endpoints:

- /chain/height
- /chain/score

Note: other chain endpoints include block and transaction objects that are fairly large.

### CODE REVIEW

File: spec/routes.yml [hunk 1]
```diff
@@ -55,3 +56,33 @@
...
+                    application/json:
+                        schema:
+                            $ref: './schemas/BlockHeightDTO.yml'
+'/chain/score':
+    get:
+        tags:
+            - Chain
+        summary: Chain Score
+        description: Gets the current scroe of the block chain.
+        operationId: chainSocre
```
Comment (Jaguar0625) [inline spec/routes.yml:81]: score

File: spec/schemas/Hash.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+type: string
+format: hex
+description: The sha3-256 hash of the last block as hex-string. an error is returned from validation.
```
Comment (Jaguar0625) [inline spec/schemas/Hash.yml:3]: "last block" - why last? why block?

File: spec/schemas/Hash.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+type: string
```
Comment (Jaguar0625) [inline spec/schemas/Hash.yml:1]: i don't see this getting used by the added routes?

File: spec/schemas/BlockHeightDTO.yml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+type: object
+description: >
+    The block height describes the position of the block within the block chain.
+    The first block of the chain has height one.
+    Each subsequent block has a height which is one higher than the previous block.
+required:
+    - height
+properties:
+    height:
+        $ref: './Height.yml'
```
Comment (Jaguar0625) [inline spec/schemas/BlockHeightDTO.yml:10]: why did you make this reference a different type + file, but ChainScoreDTO inlines the equivalent?

File: spec/schemas/Hash.yml (hunk 3 above)
Comment (fboucquez) [reply spec/schemas/Hash.yml:1]: Yea, I'll remove it and add it back in the next pr.

File: spec/schemas/BlockHeightDTO.yml (hunk 4 above)
Comment (fboucquez) [reply spec/schemas/BlockHeightDTO.yml:10]: I've created the "atomic" type ChainScore.yml, similar to Height.yml

File: spec/schemas/Hash.yml (hunk 2 above)
Comment (fboucquez) [reply spec/schemas/Hash.yml]: Hash.yml removed for now

**Review by Jaguar0625 — APPROVED** [review]

**Pull request merged**